src/test/java/com/nokia/uwr/assigntmentsalgorithm/RandomAssignmentsAlgorithmImplTest.java

```diff
@@ -0,0 +1,38 @@
+package com.nokia.uwr.assigntmentsalgorithm;
+
+import com.nokia.uwr.model.Measurements;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.Map;
+
+class RandomAssignmentsAlgorithmImplTest {
+
+    private static AssignmentsAlgorithm assignmentsAlgorithm;
+
+    @BeforeEach
```

Comment:
We don't need to run it every time the method is called. It could be @BeforeAll.
Then u need to make this method static, same with field assignmentsAlgorithm.

Reply:
Yeah, I know. It just didn't work and i did not know why. But if i understand you correctly, if I make method + field static, it should work, right?

Reply:
U do not have to make static, change only this annotation to BeforeAll and add annotation on class level: @TestInstance(TestInstance.Lifecycle.PER_CLASS)